### ISSUE TITLE
Architecture Definition and Training Details

### ISSUE THREAD

**Issue by prakharg24** [dog-qiuqiu/Yolo-Fastest#16]
Hi,

Really interesting work. I was wondering where can I find a simple architecture definition for Yolo-Fastest or Yolo-Fastest-XL? I am asking because I wanted to port it to TensorFlow to compare it with my own work.

Also, if you can provide more details on the training method, like which data augmentations, loss functions etc. you use, it would also be an interesting study.

Even if you are not releasing a paper, a simple doc which such details would be really helpful for someone following up on your work.

Thank You
